application-workloads/safekit/safekit-cluster-mirror/azuredeploy.json

```diff
@@ -47,30 +47,12 @@
 			},
 			"defaultValue": "[uniqueString(resourceGroup().id)]"
 		},
-		"azurePwsh": {
-			"type": "string",
-			"metadata": {
-				"description": "install azure powershell module (optional)"
-			},
-			"defaultValue": "no",
-			"allowedValues": [
-				"yes",
-				"no"
-			]
-		},
 		"clusterLocation": {
 			"type": "string",
 			"metadata": {
 				"description": "location of the resources (region must support Availability Zones)"
 			}
 		},
-		"location": {
```

Comment:
revert this change and use the location parameter instead of the clusterlocation - since there's only one location being used, the param should be named "location"